Source/Core/DolphinQt2/ToolBar.h

```diff
@@ -46,10 +46,10 @@ class ToolBar final : public QToolBar
 
   void MakeActions();
   void UpdateIcons();
+  void UpdatePausePlayButtonState(const bool playing_state);
```

Comment:
const here can be removed, it doesn't do anything when used by-value in a declaration. It only applies in the definition.